server/model/database.go

```diff
@@ -0,0 +1,6 @@
+package model
+
+const (
+	DBTypeMySQL    = "mysql"
```

Comment:
We really should have these constants in the `public/model` package, I was surprised to find out we didn't lol.
Don't need to change this now of course :P

Reply:
Agreed.